var/spack/repos/builtin/packages/googletest/package.py

```diff
@@ -9,13 +9,16 @@
 class Googletest(CMakePackage):
     """Google test framework for C++.  Also called gtest."""
     homepage = "https://github.com/google/googletest"
-    url      = "https://github.com/google/googletest/tarball/release-1.10.0"
     git      = "https://github.com/google/googletest"
 
     version('main', branch='main')
-    version('1.11.0', sha256='07b0896360f8e14414a8419e35515da0be085c5b4547c914ab8f4684ef0a3a8e')
-    version('1.10.0', sha256='e4a7cd97c903818abe7ddb129db9c41cc9fd9e2ded654be57ced26d45c72e4c9', preferred=True)
-    version('1.8.1',  sha256='8e40a005e098b1ba917d64104549e3da274e31261dedc57d6250fe91391b2e84')
+    version('1.11.0', url='https://github.com/google/googletest/archive/refs/tags/release-1.11.0.tar.gz',
+            sha256='07b0896360f8e14414a8419e35515da0be085c5b4547c914ab8f4684ef0a3a8e')
+    version('1.10.0', url='https://github.com/google/googletest/archive/refs/tags/release-1.10.0.tar.gz',
+            sha256='e4a7cd97c903818abe7ddb129db9c41cc9fd9e2ded654be57ced26d45c72e4c9',
+            preferred=True)
+    version('1.8.1',  url='https://github.com/google/googletest/archive/refs/tags/release-1.8.1.tar.gz',
+            sha256='8e40a005e098b1ba917d64104549e3da274e31261dedc57d6250fe91391b2e84')
     version('1.8.0',  sha256='d8c33605d23d303b08a912eaee7f84c4e091d6e3d90e9a8ec8aaf7450dfe2568')
```

Comment:
@wyphan - how will version 1.6-1.8 determine the url if the package level url has been removed?

Reply:
That's a good question. For some reason I can't find them in the GitHub releases page. It only goes as far back as v1.8.1. Or should I check the tags section to see whether they have it there?